src/algorithm/branching.jl

```diff
@@ -174,6 +176,85 @@ function branching_score(
     return VariableBranchingScore(f⁻, f⁺, min(f⁻, f⁺))
 end
 
+function branching_score(
+    state::CurrentState,
+    form::DMIPFormulation,
+    bc::VariableBranchingCandidate,
+    parent_result::NodeResult,
+    parent_node::Node,
+    config::AlgorithmConfig{StrongBranching},
+)
+    sb_state = config.branching_rule.strong_branching_state
+    if !config.branching_rule.init
```

Comment:
tbh I don't think we need to keep a persistent `strong_branching_state` in the algorithm configuration. We can just rebuild from scratch each time we want to do strong branching. It's more inefficient than what you have here, but simpler and I doubt it will ever be a bottleneck.